src/template.rs

```diff
@@ -135,6 +135,13 @@ impl Template {
         }
     }
 
+    fn unescape_tags<T>(txt: T) -> String
+    where
+        T: Into<String>,
```

Comment:
Good call. I didn't see `str` [had that impl](https://doc.rust-lang.org/std/primitive.str.html#method.replace). I'll update and push that change tonight